packages/node_modules/@ciscospark/react-container-notifications/src/selectors.js

```diff
@@ -17,7 +17,7 @@ const getUnsentNotifications = createSelector(
           return Object.assign({}, notification, {
             username: notificationActivity.actor.displayName,
             message: notificationActivity.object.displayName,
-            avatar: avatars[notificationActivity.actor.id]
+            avatar: avatars.get(notificationActivity.actor.id)
```

Comment:
Does this make it return an immutable object for `avatar` instead of a js object?

Reply:
Not sure what was going through my brain when I was doing this one. Gonna revert it because this definitely doesn't make sense.